.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
We should remove this file from git. I will proceed to merg the PR anyways, but let's open clear this file from git on the next PR.

Reply:
@gvescu